app.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "cc-amend",
+  "description": "Compile batch results for codeclimate test coverage",
+  "repository": "https://github.com/grosser/cc-amend",
+  "keywords": ["codeclimate", "codeclimate_batch", "travis"],
+  "addons": [
+    "memcachier:dev"
+  ],
+  "env": {
+    "CUSTOM_RUBY_VERSION": "2.2.4",
+    "CODECLIMATE_REPO_TOKEN": "XXXXXX",
+    "ROLLBAR": "Rollbar key",
+    "TO_FILE": "1"
```

Comment:
there is a `CODECLIMATE_TO_FILE ` option from https://github.com/codeclimate/ruby-test-reporter/pull/84